.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  pull_request:
+  push:
+
+jobs:
+  lint:
```

Comment:
Suggestion: do we want to call this something other than `lint` just to clarify that it's also running unit tests and static analysis?

Reply:
Ah yes, we probably do